libr/core/cmd_project.c

```diff
@@ -57,9 +57,12 @@ static int cmd_project(void *data, const char *input) {
 		if (!file || !file[0]) { /* if no argument specified use current project */
 			file = str;
 		}
+		r_config_set (core->config, "prj.name", file);
```

Comment:
This var should be set inside project_save maybe, so you can remove all that logic from here